### ISSUE TITLE
Query plugin used to read "eventsByTag" from the journal is complaining about Bad Request

### ISSUE THREAD

**Issue by agregoris-zd** [j5ik2o/akka-persistence-dynamodb#64]
When I've tried to use the "eventsByTag" feature in the Query plugin, I found an error with the following log message:

```
[info] [2020-02-04 13:26:34,641] [WARN] [com.github.j5ik2o.akka.persistence.dynamodb.query.JournalSequenceActor] [bla-akka.actor.default-dispatcher-47] [akka://bla@127.0.0.1:25520/system/j5ik2o.dynamo-db-read-journal.akka-persistence-dynamodb-journal-sequence-actor] - Failed to query max ordering id because of java.util.concurrent.CompletionException: software.amazon.awssdk.services.dynamodb.model.DynamoDbException: Either the KeyConditions or KeyConditionExpression parameter must be specified in the request. (Service: DynamoDb, Status Code: 400, Request ID: 6281d9df-6c6d-4f6f-b733-569e6b2cd603), retrying in 1 minute
```

Then I look at the code and I found that there is a Query in `ReadJournalDaoImpl.scala` which does not define any KeyConditions or KeyConditionExpression parameter which is required that means DynamoDB will reject all of this query as Bad Request, the code is:

```
val queryRequest = QueryRequest
            .builder().tableName(tableName).select(Select.SPECIFIC_ATTRIBUTES).attributesToGet(
              columnsDefConfig.orderingColumnName
            ).limit(queryBatchSize).exclusiveStartKeyAsScala(lastEvaluatedKey).build()
```

**Comment by FelixKuehl** [comment]
Hey, +1 on this I am getting the exact same error. Are there any news on this?

**Comment by FelixKuehl** [comment]
I Think this might come down to ReadJournalDaoImpl.scala:151
 ```
...
Source
      .single(QueryRequest.builder().tableName(tableName).build())
      .via(streamClient.queryFlow(parallelism))
...
```

Maybe this was intended to be a scan request / flow? Changing it to
 ```
...
Source
      .single(ScanRequest.builder().tableName(tableName).build())
      .via(streamClient.scanFlow(parallelism))
...
```
Should in theory work. (Not tested)

**Comment by j5ik2o** [comment]
The reproducibility of this bug was confirmed. Please give me time to fix the bug.

**Comment by j5ik2o** [comment]
You can try the bug fixes included in the branch by following the steps below.

```scala
resolvers += "jitpack" at "https://jitpack.io"
libraryDependencies += "com.github.j5ik2o" % "akka-persistence-dynamodb" % "feature~fixJournalSequence-SNAPSHOT"
```

**Comment by j5ik2o** [comment]
fix it in v1.0.21
https://github.com/j5ik2o/akka-persistence-dynamodb/releases/tag/v1.0.21
